src/scss/menu/common.scss

```diff
@@ -67,6 +67,12 @@
 			.tab:hover, .tab.active { color: $colorBlack; }
 			.tab:last-child { margin: 0px; }
 		}
+
+		.menuTitleWrapper { display: flex; gap: 0 6px; padding: 8px 14px; align-items: center; }
```

Comment:
Hey, just curious, is there a reason to have these separate instead of combined like this:

```scss
.menuTitleWrapper {
	display: flex; gap: 0 6px; padding: 8px 14px; align-items: center; 

	.icon.back { width: 20px; height: 20px; background-image: url('~img/arrow/menu.svg'); transform: rotateZ(180deg); cursor: pointer; }
	.title { @include text-common; font-weight: 600; color: $colorBlack; padding: 0; }
}
```

Reply:
yes, it's a lot more readable

Reply:
Hello! Yes, we use such code-style in order to increase readability

Reply:
Oh - I see, never saw that format in css yet so I might not "used" to it yet 😬